source/manual/create-a-gpg-key.html.md

```diff
@@ -54,7 +54,7 @@ gpg --send-keys $KEYID
 ```
 If you are having problems uploading your key, it's worth trying another keyserver. Ultimately the uploaded key will be synced to the other keyservers so it doesn't matter which one you chose.
 
-You now should be able to find your key on <http://keys.gnupg.net/>
```

Comment:
You might want to consider adding the port 11371 to the docs, so http://keys.gnupg.net:11371/ . I forget the details of how SKS (the bit of software most if not all of the keyservers will be running) works, but this seems to help to access the web interface.